packages/core/src/components/forms/controls.tsx

```diff
@@ -52,7 +52,7 @@ export class Control<P extends IControlProps> extends React.Component<React.HTML
         return (
             <label className={className} style={this.props.style}>
                 <input
-                    {...removeNonHTMLProps(this.props, ["children", "indeterminate"], true)}
+                    {...removeNonHTMLProps(this.props, ["children"], true)}
```

Comment:
ok now i'm wondering if `children` should be in `INVALID_PROPS`? 

it's not an HTML prop, though it is supported by JSX, which is why it has to be removed here. we're handling `this.props.children` ourselves cuz this is React.